data/tribes/initialization/atlanteans/units.lua

```diff
@@ -1473,45 +1496,80 @@ descriptions:new_tribe {
             -- TRANSLATORS: Purpose helptext for an atlantean production site: Hunter's House
             purpose = pgettext("building", "Hunts animals to produce meat."),
             -- TRANSLATORS: Note helptext for an atlantean production site: Hunter's House
-            note = pgettext("atlanteans_building", "The hunter’s house needs animals to hunt within the work area.")
+            note = pgettext("atlanteans_building", "The hunter’s house needs animals to hunt within the work area."),
+            -- TRANSLATORS: Lore helptext for an atlantean production site: Hunter's House
+            lore = pgettext("atlanteans_building", "Sometimes we end up in a region with no fish, so we must hunt, "..
+                                                   "Game will reproduce by itself, if not hunted to much. But yes Fish is our main meal."),
+            -- TRANSLATORS: Lore author helptext for an atlantean production site: Hunter's House
+            lore_author = pgettext("atlanteans_building", "Hunter answering a Fisher")
          }
       },
       {
          name = "atlanteans_well",
          helptexts = {
             -- TRANSLATORS: Purpose helptext for an atlantean production site: Well
-            purpose = pgettext("building", "Draws water out of the deep.")
+            purpose = pgettext("building", "Draws water out of the deep."),
+            -- TRANSLATORS: Note helptext for an atlantean production site: Well
+            note = pgettext("atlanteans_building", "I hope you checked this spot with a gelogist."),
+            -- TRANSLATORS: Lore helptext for an atlantean production site: Well
+            lore = pgettext("atlanteans_building", "Well, there is no well being without a well."),
```

Comment:
Please easy on puns... Think of poor translators. :)

Reply:
Der Typ an der Quell‘
der merkt ganz schnell
bin ein armer Gesell
ohne die Quell‘!

Reply:
:+1:

*De Keerl bi de Well
he markt daar ganz fell
hüm gaht dat leep grell
sünner sien' Well!*

And if Hungarian is harder, just translate it more freely ;)

Reply:
I just asked to not overdo them. A few here and there are OK. :)
Rhymes also, though they are almost required for lore texts, so I guess all translators just get to feel the pain of the original authors for these. :)))

BTW it should be either wellbeing or well-being: https://grammarist.com/spelling/well-being-wellbeing/

Reply:
```suggestion
            lore = pgettext("atlanteans_building", "Well, there is no wellbeing without a well."),
```

Reply:
> And if Hungarian is harder, just translate it more freely ;)

"Aki az után kutat, hogy hol ásson kutat, az egy kútkutató? Jó lenne egy útmutató…" :P